src/main/webapp/app/tasks/default-task.service.ts

```diff
@@ -23,4 +23,11 @@ export class DefaultTaskService implements TaskService {
   getAll(): Observable<Task[]> {
     return this.http.get<Task[]>(this.baseUrl + '/tasks');
   }
+
+  updateStatus(id: string, status: TaskStatus): Observable<Task> {
+    return this.http.put<Task>(this.baseUrl + '/tasks/' + id + '/status', { status });
+  }
+  clearDoneTasks(): Observable<Task> {
+    return this.http.put<Task>(this.baseUrl + '/tasks/clear', { status });
```

Comment:
This would probably be a DELETE call.

Reply:
Nice finding! Yes, you are right, I knew it but forgot it. so a `DELETE` HTTP verb should be used when we are deleting a resource 👍